src/sentry/ratelimits/redis.py

```diff
@@ -1,24 +1,25 @@
 from time import time
 
+from django.conf import settings
 from redis.exceptions import RedisError
 from sentry_sdk import capture_exception
 
 from sentry.exceptions import InvalidConfiguration
 from sentry.ratelimits.base import RateLimiter
+from sentry.utils import redis
 from sentry.utils.hashlib import md5_text
-from sentry.utils.redis import get_cluster_from_options
 
 
 class RedisRateLimiter(RateLimiter):
     window = 60
 
     def __init__(self, **options):
-        self.cluster, options = get_cluster_from_options("SENTRY_RATELIMITER_OPTIONS", options)
+        cluster_key = getattr(settings, "SENTRY_RATE_LIMIT_REDIS_CLUSTER", "default")
+        self.client = redis.redis_clusters.get(cluster_key)
 
     def validate(self):
```

Comment:
I don't actually see any calls to this at all, I think it can be removed

Reply:
The validate function is a service class thing iirc